docs/journalist/sources.rst

```diff
@@ -39,24 +39,26 @@ Sending a reply
 ---------------
 
 Compose a reply to the selected source in the text box at the bottom of the
-conversation view. You may send it, or leave it in the box without sending to
-save it as a draft.
+conversation view. Click the paper airplane icon or press "Ctrl+Enter" to send
+a reply. Any replies you did not send will be discarded when you exit the
```

Comment:
Maybe: "Any unsent replies"

Reply:
This is where it gets a bit tricky. If the user submitted a reply but it _failed_ to send, it will not be discarded -- the failed message will still be visible on the next client start. That's why I went with "you did not send", but let me know if you have other ideas in this regard.